chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/rules/developer/DeveloperPriorStatusActiveValidation.java

```diff
@@ -0,0 +1,41 @@
+package gov.healthit.chpl.manager.rules.developer;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import gov.healthit.chpl.dto.DeveloperDTO;
+import gov.healthit.chpl.dto.DeveloperStatusEventDTO;
+import gov.healthit.chpl.entity.developer.DeveloperStatusType;
+import gov.healthit.chpl.manager.rules.ValidationRule;
+import gov.healthit.chpl.permissions.ResourcePermissions;
+import gov.healthit.chpl.util.AuthUtil;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+
+public class DeveloperPriorStatusActiveValidation extends ValidationRule<DeveloperValidationContext> {
+    private static final Logger LOGGER = LogManager.getLogger(DeveloperPriorStatusActiveValidation.class);
+    private ResourcePermissions resourcePermissions;
+
+    public DeveloperPriorStatusActiveValidation(final ResourcePermissions resourcePermissions) {
+        this.resourcePermissions = resourcePermissions;
+    }
+
+    /**
+     * If the before status is not Active and the user is not ROLE_ADMIN then
```

Comment:
ROLE_ONC is missing from this comment. Suggest removing the comment entirely

Reply:
Left as is as figured writer of the impl knew better than me. I lke the idea of fixing the comment (adding ROLE ONC) but since there is a property referenced and one can look up the error, I'll ditch it as requested.